src/main/java/com/google/firebase/auth/ListUsersPage.java

```diff
@@ -237,17 +238,17 @@ String getNextPageToken() {
    * before attempting to load any user data (which is expensive, and hence may be performed
    * asynchronously on a separate thread).
    */
-  static class PageFactory {
+  static class UserPageFactory {
```

Comment:
May be just call this `Factory` and use the full qualified name of `ListUsersPage.Factory` at callsites?

Reply:
Good idea!